filters/HAGFilter.cpp

```diff
@@ -195,6 +196,7 @@ void HAGFilter::addArgs(ProgramArgs& args)
 {
     args.add("count", "The number of points to fetch to determine the "
         "ground point [Default: 1].", m_count, point_count_t(1));
+    args.add("raster", "GDAL-readable raster to use for DEM (uses band 1)", m_rasterName, "NONE");
```

Comment:
I don't see any reason to use a sentinel value here ("NONE").  An empty string seems better.

Reply:
Yeah good point, fixing now.